po/de.po

```diff
@@ -7736,6 +7743,9 @@ msgstr "Mörsernachladezeit -10%"
 msgid "Mortar Targeting Computer"
 msgstr "Mörser-Zielcomputer"
 
+# In data/base/messages/strings/resstrings.txt ist MRL eimal mit "Mini-Rocket Battery" und einmal
+# mit "Mini-Rocket Artillery" angegeben. Daher kann meiner Meinung nach trotz des Kommentars von Forgon
+# die Übersetzung so bleiben wie sie ist.
```

Comment:
MRL steht vermutlich für "[Multiple Rocket Launcher](https://en.wikipedia.org/w/index.php?title=Multiple_rocket_launcher&oldid=888380814)", was im Deutschen schlicht ein "[Mehrfachraketenwerfer](https://de.wikipedia.org/w/index.php?title=Multiple_Launch_Rocket_System&oldid=186433301)" wäre. Vielleicht soll es in Warzone 2100 ja für "Mini-Rocket Launcher" stehen?

Reply:
Ist eine logische Annahme. Ich stelle es Dir frei, ob Du jetzt daraus dann einen "Miniraketenwerfer" machen willst oder ob Du die derzeitige Übersetzung beibehälst. Mir ist es egal.

Reply:
Dann bleibt die Übersetzung so wie sie ist. IIRC ist das auch im Original so.